src/content/developers/docs/scaling/validium/index.md

```diff
@@ -7,30 +7,64 @@ incomplete: true
 sidebarDepth: 3
 ---
 
-Uses validity proofs like [ZK-rollups](/developers/docs/scaling/zk-rollups/) but data is not stored on the main layer 1 Ethereum chain. This can lead to 10k transactions per second per validium chain and multiple chains can be run in parallel.
+Validium is a [scaling solution](/developers/docs/scaling/) that enforces integrity of transactions using validity proofs like [ZK-rollups](/developers/docs/scaling/zk-rollups/), but doesn’t store transaction data on the main Ethereum layer. While off-chain data availability introduces trade-offs, it can lead to massive improvements in scalability (validiums can process ~ 9,000 transactions, or more, per second). 
```

Comment:
> doesn’t store transaction data on the main Ethereum layer

This reads a bit vague to me - should we be explicit to say "doesn't store transaction data on chain"?

Reply:
I felt saying validiums don't store data "on-chain" at the beginning of the page would throw off readers ("what chain are we referring to?"). Establishing the main point (i.e., validiums don't store data on Ethereum) from the start (as opposed to ZK-rollups) was the priority here. Also, I think the idea of on-chain vs off-chain storage is well discussed in the other sections, so readers are likely to understand the concept.

Reply:
I think we can be fairly explicit and accurate here imo. I would argue the page we're editing is in the "Advanced" section of the docs, and if we're going to refer to where the data is being stored, I would rather we use proper terms like "Mainnet execution layer" or just "Ethereum Mainnet" over "main Ethereum layer" for example.
I think using "on-chain" is fine, but to your point if you want to clarify we're referring to Ethereum Mainnet here I think that's perfectly reasonable.

Reply:
```suggestion
Validium is a [scaling solution](/developers/docs/scaling/) that enforces integrity of transactions using validity proofs like [ZK-rollups](/developers/docs/scaling/zk-rollups/), but doesn’t store transaction data on the Ethereum Mainnet. While off-chain data availability introduces trade-offs, it can lead to massive improvements in scalability (validiums can process ~ 9,000 transactions, or more, per second). 
```